exercises/ruby-1-a/README.md

```diff
@@ -4,17 +4,18 @@ This is Part 1 of our research into how people write Ruby differently. This exer
 
 ## Instructions
 
-Your task is to determine which word in a sentence has the most vowels (a, e, i, o, u). A sentence is defined as a [Ruby string][docs-string] consisting of one or more words. A word consists of one or more upper- or lowercase letters. Words are separated from each other by a single space character.
-
-Your job is to define a method with the signature `word_with_most_vowels`, which takes a sentence, and returns the word with the most vowels.
+Your task is to define a `ShoppingCart` class that can receive a hash of items where the key is the name of the item and the value the price of the item, see the example below. The `ShoppingCart` class will also have two methods: `total_amount` and `items_list`. The method `total_amount` will return the sum of all the items’ prices. The method `list` will return a string with the name of the items separated by commas.
```

Comment:
Could we add a little overview before digging into the technical spec?

Something like:

> Your task is to develop a shopping cart, which can contain items and calculate total cart price. This will be modelled as `ShoppingCart` class...

Reply:
>  The method `list` 

Should this be `items_list`?

Reply:
Thanks for your feedback. I added a small overview and fixed the method name